cmake/Corrosion.cmake

```diff
@@ -1092,6 +1092,25 @@ function(corrosion_link_libraries target_name)
     endforeach()
 endfunction()
 
+#[=======================================================================[.md:
+ANCHOR: corrosion-install
+```cmake
+corrosion_install(TARGETS <target1> ... <targetN> [EXPORT <export-name>]
+                  [[ARCHIVE|LIBRARY|RUNTIME|PRIVATE_HEADER|PUBLIC_HEADER]
+                   [DESTINATION <dir>]
+                   [PERMISSIONS <permissions...>]
+                   [CONFIGURATIONS [Debug|Release|<other-configuration>]]
+                  ] [...])
+```
+* **TARGETS**: Target or targets to install.
```

Comment:
I think we should clarify here that the targets must have been targets added by corrosion. Perhaps something like:

> Rust library or executable targets imported by corrosion

Reply:
I *think*, but I'm not certain, that we'll need to accept all target kinds, since if the users want to create C++ bindings as well they will need to create a new target that depends on the rust lib target. Then in `install(TARGETS rust-lib cpp-bindings EXPORT MyProjectTargets)` we need to list all targets and their dependents, otherwise it will error.